.travis.yml

```diff
@@ -42,6 +44,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   # Install dependencies for source repos
   - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
+  - export PYTHONPATH=/usr/lib/python2.7/dist-packages:$PYTHONPATH
```

Comment:
You can avoid this by using `language: generic` with cpp compiler, as in https://github.com/ros-industrial/industrial_ci/blob/master/.travis.yml